src/bootstrap/bootstrap.py

```diff
@@ -597,10 +597,8 @@ def build_bootstrap(self):
                 self.cargo()))
         args = [self.cargo(), "build", "--manifest-path",
                 os.path.join(self.rust_root, "src/bootstrap/Cargo.toml")]
-        if self.verbose:
+        for _ in range(0, max(0, self.verbose - 1)):
```

Comment:
In both Python 2 and 3, `range(0, self.verbose - 1)` will be empty when `self.verbose - 1` is zero or negative, so the `max(0, _)` is unnecessary.